src/pages/index.js

```diff
@@ -1,10 +1,15 @@
+// 📁 index.js
+
 import './index.css';
+import Api from '../utils/Api.js';
+import { setButtonText } from '../utils/helpers.js';
 import { enableValidation } from '../scripts/validation.js';
```

Comment:
* Validation doesn't work

Please, call `enableValidation` in `index.js`, to enable validation